docs/index.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <script src="assets/index.js"></script>
+  <link rel="stylesheet" href="assets/styles.css">
+  <link href="https://fonts.googleapis.com/css?family=Open+Sans|Roboto+Mono&display=swap" rel="stylesheet">
+  <title>Snippets</title>
+</head>
+<body>
+  <div class="container">
+    <h1>Snippets</h1>
+    <p>List of available snippets</p>
+    <div class="snippets-container">
+      <ul id="snippetsContainer">
```

Comment:
Ideally, this would be something that would expand on clicking. That should also show the description and other details

Reply:
Need not be addressed as a part of this PS